tests/units/Extension/Classes.php

```diff
@@ -0,0 +1,713 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Wasm\Tests\Units\Extension;
+
+use ArrayAccess;
+use Exception;
+use ReflectionClass;
+use ReflectionExtension;
+use ReflectionMethod;
+use WasmArrayBuffer;
+use WasmInt16Array;
+use WasmInt32Array;
+use WasmInt8Array;
+use WasmUint16Array;
+use WasmUint32Array;
+use WasmUint8Array;
+use Wasm\Tests\Suite;
+
+class Classes extends Suite
+{
+    public function getTestedClassName()
+    {
+        return 'StdClass';
```

Comment:
Consider turning any referenced class into `foo::class` rather than using the string notation: makes the intent more explicit

Reply:
True!